package.json

```diff
@@ -135,6 +135,7 @@
     "style-loader": "^0.23.1",
     "swagger-ui-dist": "^4.13.2",
     "swagger-ui-express": "^4.5.0",
```

Comment:
I added tough cookie after I got the following message installing `parse/node-apn`:

```
 $ npm install parse-community/node-apn
npm WARN axios-cookiejar-support@4.0.6 requires a peer of tough-cookie@>=4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN http-cookie-agent@5.0.4 requires a peer of tough-cookie@^4.0.0 but none is installed. You must install peer dependencies yourself.

+ @parse/node-apn@5.1.3
added 14 packages from 15 contributors and audited 1110 packages in 465.071s
```